src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -126,9 +127,8 @@ public UserTenant save(UUID consortiumId, UserTenant userTenantDto) {
 
     Optional<UserTenantEntity> userTenant = userTenantRepository.findByUserIdAndIsPrimary(userTenantDto.getUserId(), IS_PRIMARY_TRUE);
     if (userTenant.isEmpty()) {
-      log.warn("Could not find user with id: {} in user_tenant table for tenant id: {}",
-        userTenantDto.getUserId(), userTenantDto.getTenantId());
-      throw new ResourceNotFoundException(USER_ID, String.valueOf(userTenantDto.getUserId()));
+      log.warn("Could not find user '{}' with primary affiliation in user_tenant table", userTenantDto.getUserId());
+      throw new ResourceNotFoundException(String.format(NOT_FOUND_PRIMARY_AFFILIATION_MSG,USER_ID, userTenantDto.getUserId()));
```

Comment:
missed space

Reply:
sorry, I didn't catch it. could you please explain a little bit

Reply:
fixed